principles/index.html

```diff
@@ -64,22 +55,84 @@ <h2>How This Document Fits In</h2>
-      </p>
-      <p>
-        Indeed, when the cross-context sharing of identifiers and of behavioural data is involved, the evidence
-        shows that no consent mechanism will support people in making autonomous choices.
-      </p>
-      <div class="issue">
-        <p>
-          Return to this with greater detail.
+      <h3>Measurement</h3>
+
+      <section>
+        <h4>Measurement should be private for safe, widespread usage, but always be under user control</h4>
+      </section>
+
+      <section>
+        <h4>Opting-out should not be visible</h4>
+        <p>Users may wish to opt-out of participation in measurement, but do so in such a way that is not visible to the sites they visit. Visible opt-out could lead to retaliation against, or coercion of, users who do not wish to participate in measurement.</p>
+      </section>
+
+      <section>
+        <h4>Measurement should not significantly enable cross-context recognition</h4>
+        
+        <p>Protections of differential privacy take the form of guarantees that aggregation or noise make participation in a particular measurement mostly indistinguishable, but also recognize that some information (as often quantified by parameters including epsilon) is released and could be combined with other known information to learn something with some (presumably very small) probability. "significantly" here is not yet detailed. The aggregated or noised measurement should not reasonably be usable to identify a particular user or to link an user's activity to another context.</p>
+
+        <p class="note">
+          Metrics to define significance are being evaluated by a separate task force.
         </p>
-      </div>
+
+        <p>Because measurement and attribution involve all kinds of viewing advertisements and a variety of other actions, in a wide range of different contexts, relying on understanding of, expectations about and consent over cross-context recognition as a result of ad measurement would be inappropriate.</p>
+      </section>
+
+      <section>
+        <h4>Measurement should not significantly enable inferences about individual people from their participation in the measurement</h4>
+
+        <p>Related to cross-context recognition, measurement mechanisms should not reasonably be able to be used to learn or infer information about a particular user, for example, that a user visited a site (or class of site) or took an online or offline action.</p>
+
+        <p>Population-level measurement can still be used for inference; this principle only indicates that participation (or non-participation) in the measurement cannot be used to enable an inference about that individual.</p>
+      </section>
+    </section>
+    <section>
+      <h3>Accountability</h3>
+      <section>
+        <h4>Users should be able to investigate how data about them is used and shared.</h4>
+
+        <p>Users should be able to learn what measurements they participate in.</p>
```

Comment:
```suggestion
        <p>Users should be able to learn what measurements they may participate in.</p>
```

In nearly all these proposals, there's no guarantee the measurement actually happens, so the best we can likely do is state which measurements they may participate in.